### DISCUSSION TITLE
version 2.0 beta on website still 1.7?

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1162] [General]
While you recently made the decision to turn 1.7 in 2.0, this is not reflected on the website in the downloads. Is this intentional or have you simply forgotten to update the website to reflect the change?

**Comment by vkbo** [comment]
I cannot change a pre-release that has already been made, so 1.7 beta 1 will remain there until the next pre-release, which will be 2.0 RC 1.
